accelerometer/accUtils.py

```diff
@@ -147,12 +147,19 @@ def writeStudyAccProcessCmds(accDir, outDir, cmdsFile='processCmds.txt',
     nonWearDir = os.path.join(outDir, 'nonWear')
     stationaryDir = os.path.join(outDir, 'stationary')
     logsDir = os.path.join(outDir, 'clusterLogs')
+    rawDir = os.path.join(outDir, 'raw') 
+    npyDir = os.path.join(outDir, 'npy') 
+    outputDir = os.path.join(outDir) 
```

Comment:
This bit is not needed. `os.path.join` is for joining two path strings, e.g. `os.path.join('a/b', 'c')` will produce the string `'a/b/c'`. You get the same result by simply doing `'a/b' + '/' + 'c'`, but using `os.path.join` is safer because it handles trailing slahes `'/'` automatically (so `os.path.join('a/b/', 'c')` will also produce correctly `'a/b/c'`).

Reply:
So here you can simply remove this line, and use `outDir` directly below instead of `outputDir`